### PR TITLE
Add missing sequences for passing unified tests

### PR DESCRIPTION
**Pull request by Tatarinho**

### CODE REVIEW

**Review by MichalCz — APPROVED** [review]

**Comment by jan-warchol** [comment]
A general question (I know it's a bit late to suggest that, but maybe for the future): what about using plain JS in reference apps whenever they don't test something typescript-specific? this would make the deployment easier (no build step for an app).

File: js/topic-consumer/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console, no-extra-parens */
+import { Streamable, TransformApp } from "@scramjet/types";
+import { StringStream } from "scramjet";
+import { PassThrough } from "stream";
+
+const mod: (TransformApp | { requires: string, contentType: string})[] = [
+    { requires: "pytopic-test", contentType: "text/plain" },
+    function(input: Streamable<any>) {
+        const out = new PassThrough({ objectMode: true });
```
Comment (jan-warchol) [inline js/topic-consumer/src/index.ts:9]: interesting, why do we need `objectMode`?

File: js/topic-consumer/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        (input as StringStream)
+            .map((s: any) => `consumer got: ${s}\n`)
+            .pipe(out);
+
+        return out;
+    }
+];
```
Comment (jan-warchol) [inline js/topic-consumer/src/index.ts:16]: why wrap this function in an array? there's only one.

File: js/topic-producer/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console, no-extra-parens */
+import { Streamable, TransformApp } from "@scramjet/types";
+import { StringStream } from "scramjet";
+import { PassThrough } from "stream";
+
+const mod: (TransformApp | { provides: string, contentType: string})[] = [
```
Comment (jan-warchol) [inline js/topic-producer/src/index.ts:6]: why wrap this function in an array? there's only one.

File: js/unhealthy/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+const mod = async function(_stream) {
+    this.addMonitoringHandler(() => {
+        return { healthy: false };
+    });
+
+    for (var member in this){
+        console.log("////", member);
+    }
+
```
Comment (jan-warchol) [inline js/unhealthy/src/index.ts:11]: Is this some leftover debug?

File: python/debug-args/args.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from scramjet.streams import Stream
+import json
+
+def run(context, input, named_arg, number_arg):
+    output = {"named_arg": named_arg, "number_arg": number_arg}
```
Comment (jan-warchol) [inline python/debug-args/args.py:5]: ditto for arg descriptions

**Review by jan-warchol — COMMENTED** [review]
generally LGTM apart from a few minor things.

**Pull request merged**